docs/source/overview_software_architecture.rst

```diff
@@ -176,8 +176,8 @@ TypeMap
 BuildManager
 ^^^^^^^^^^^^
 
-* Responsible for memoizing ``Builders`` and ``Containers``
```

Comment:
"memoizing" is actually the proper term here I believe

Reply:
ok, i am no native speaker, my background for this is the latin word memorare, which is with an "r" and means to remember sth. But the english language is not very straight forward in its spelling so I could be mistaking.
[memoizing](https://www.dict.cc/?s=memoizing) imho would mean, something like making a memo out of it, whereas [memorizing](https://www.dict.cc/?s=memorizing) would mean to remember it.

Reply:
How we say in germany: "Again what learnt"
I added the external wikipedia link to the word in the docu.